frontend/projects/valtimo/layout/src/lib/components/widget-management/management-content/external-plugin/widget-management-external-plugin.component.scss

```
@@ -0,0 +1,40 @@
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.valtimo-widget-management-external-plugin {
+  display: flex;
+  flex-direction: column;
+  gap: var(--cds-spacing-05);
+
+  ::ng-deep .cds--text-input,
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)